osmotestutils/cli_helpers.go

```diff
@@ -12,3 +14,36 @@ func DefaultFeeString(cfg network.Config) string {
 	feeCoins := sdk.NewCoins(sdk.NewCoin(cfg.BondDenom, sdk.NewInt(10)))
 	return fmt.Sprintf("--%s=%s", flags.FlagFees, feeCoins.String())
 }
+
+var (
+	base   = 10
+	bitlen = 64
+)
+
+func ParseUint64SliceFromString(s string, seperator string) ([]uint64, error) {
+	var ids []uint64
```

Comment:
Suggest changing the name from `ids`  to something like `parsedInts` since the method parses generic numbers, not necessarily ids